### ISSUE TITLE
face-mask implementation

### ISSUE THREAD

**Issue by inandi** [justadudewhohacks/face-api.js#620]
Hi,

first of all, thank you for this awesome plugin...cheers!!!

I just want to know, how I can detect face with a mask or not? do i need to create new model for this. I am trying to understand the manifest JSON files and shard files. can you give me some tips!!!

**Comment by m-alcu** [comment]
i'm in the same wave... I suppose that it is necessary to train a net similar to face expression or perhaps swap this net with a facemask net....

How to train a new net?

**Comment by luanfonceca** [comment]
+1 to this issue. Did you guys were able to figure that out?

Thanks!

**Comment by rm5565** [comment]
+1. Also looking for a face-with-mask detection solution with face-api.js.   Would be great.

**Comment by raulocho** [comment]
Hi there i have a facemask model trained, now just trying figure how i setup that into faceapi for concat the results. Any help will be welcome guys.

Great proyect thank you for the efford

**Comment by aveloso4** [comment]
@raulocho Did you already managed to setup your trained model into faceapi?

**Comment by raulocho** [comment]
@aveloso4 Yes exactly, sorry for delay answering, had a bad time that weeks with work. I have the model ready just need take the face from face api and concat the result in the analysis Json.

**Comment by Reaper622** [comment]
@raulocho  Can u show some code about how to use your own mask model with face-api.js ?
